src/Application/Borrows/Queries/GetAllBorrowRequestsPaginated.cs

```diff
@@ -93,6 +70,24 @@ public QueryHandler(IApplicationDbContext context, IMapper mapper)
                 .ThenInclude(t => t.Room)
                 .ThenInclude(s => s.Department);
 
+            if (request.CurrentUser.Role.IsEmployee())
+            {
+                borrows = borrows.Where(x => x.CreatedBy! == request.CurrentUser.Id);
```

Comment:
where is the check EmployeeId == CurrentUser.Id? You removed it?

Reply:
fixed at https://github.com/ProFile-org/ProFile-BE/pull/309/commits/c41717494ebcf25bc46e19be089faa4a1539a994